### ISSUE TITLE
Quest: Add House of dApps & DeFiance to the wiki

### ISSUE THREAD

**Issue by luxumbra** [MetaFam/metagame-wiki#283]
As a team effort, we got these long overdue posts ready for the wiki. Is there someone who has some Git knowledge and is a relatively new player who could add these to the wiki this weekend? See the [discord post for details of the quest](https://discord.com/channels/629411177947987986/782054816926400533/866064029016522782)

The posts:
[House of DeFiance](https://www.notion.so/House-of-DeFiance-3122d2ba02e14159996c8a43ee01d131q)
[House of dApps](https://www.notion.so/House-of-dApps-2d77a931c57a4b3180212bca0f08e3c0)

Assign to yourself if you're going for the [quest I posted in Discord](https://discord.com/channels/629411177947987986/782054816926400533/866064029016522782)

**Comment by Baconaetor** [comment]
Well, this is taking longer than anticipated. . . I thought it was gonna be a simply copy paste operation. I'll come back to it tomorrow eve. 

I've got the House of Dapps mostly fixed up, but need to give it a content review. House of DeFi still needs some fix'ns. I deleted the sub-stack sub button as it was throwing errors and none of the other pages have one. I also changed the icon for House of Ethereum to another character that renders on more systems.
